cogs/admin.py

```diff
@@ -107,25 +101,28 @@ async def reloadutils(self, ctx, name: str):
         except Exception as e:
             error = default.traceback_maker(e)
             return await ctx.send(f"Module **{name_maker}** returned error and was not reloaded...\n{error}")
-        await ctx.send(f"Reloaded module **{name_maker}**")
+        await ctx.send(f"ℹ Reloaded module **{name_maker}**")
 
     @commands.command()
     @commands.check(permissions.is_owner)
     async def reboot(self, ctx):
         """ Reboot the bot """
-        await ctx.send("Rebooting now...")
+        await ctx.send(":warning: Rebooting now...")
```

Comment:
I recommend using the raw emojis instead here: ⚠️

Reply:
I didn't know that so many raw emojis existed, will do sir